test/integration/garbagecollector/garbage_collector_test.go

```diff
@@ -242,15 +242,15 @@ func setupWithServer(t *testing.T, result *kubeapiservertesting.TestServer, work
 		t.Fatalf("failed to create dynamicClient: %v", err)
```

Comment:
Why we need both metadataclient and dynamic client? I am curious about it.

Reply:
Looks like community hasn't migrated their integration test from dynamic client to metadata client. It is probably intended for backward compatibility. As long as KCM is fully free of dynamic client, I am ok having dynamic client in integration tests.

Reply:
https://github.com/kubernetes/kubernetes/blob/master/test/integration/garbagecollector/garbage_collector_test.go